### PR TITLE
Create members group when creating org

### PR DESCRIPTION
**Pull request by julienlancelot**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/organization/DefaultGroupCreator.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.server.organization;
+
+import org.sonar.db.DbSession;
+import org.sonar.db.user.GroupDto;
+
+public interface DefaultGroupCreator {
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/organization/DefaultGroupCreator.java]: CRITICAL Move constants to a class or enum. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1214)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* ![CRITICAL][CRITICAL] 1 critical

Watch the comments in this conversation to review them.

[CRITICAL]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL'

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [usergroups](https://github.com/SonarSource/sonarqube/blob/3647a729528cb22579dce7bd3b63804742d244d0/server/sonar-server/src/main/java/org/sonar/server/usergroups): Add a 'package-info.java' file to document the 'usergroups' package [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: server/sonar-db-dao/src/main/java/org/sonar/db/permission/GroupPermissionDao.java [hunk 2]
```diff
@@ -65,6 +65,14 @@ public int countGroupsByQuery(DbSession dbSession, PermissionQuery query) {
   }
 
   /**
+   * Select global and project permissions of a given group (Anyone group is NOT supported)
+   * Each row returns a {@link GroupPermissionDto}
+   */
+  public void selectAllPermissionsByGroupId(DbSession dbSession, String organizationUuid, Integer groupId, ResultHandler resultHandler) {
```
Comment (julienlancelot) [inline server/sonar-db-dao/src/main/java/org/sonar/db/permission/GroupPermissionDao.java]: As discussed offline, I'll keep using ResultHandler in order to stream the result

**Pull request merged**